plugins/BuildFile.xml

```diff
@@ -0,0 +1,6 @@
+<use name="FWCore/Framework"/>
+<use name="FWCore/PluginManager"/>
+<use name="FWCore/ParameterSet"/>
+<use name="cp3_llbb/Framework"/>
+<use name="cp3_llbb/TreeWrapper"/>
```

Comment:
Sorry I forgot sth here: I think you'll need a line `<use name="cp3_llbb/ZAAnalysis"/>`...